README.md

```diff
@@ -22,23 +23,54 @@ Or install it yourself as:
 
 ## Usage
 
-    validates_zipcode :zipcode
+### With ActiveModel::Validations
+
+```ruby
+validates_zipcode :zipcode
 
-    validates :zipcode, zipcode: true
+validates :zipcode, zipcode: true
+```
 
 ``ValidatesZipcode`` expects the model to have an attribute called ``country_alpha2`` to contain the country code.
 You can provide your own country_code using ``:country_code`` option, or specify which attribute contains this information
 using ``:country_code_attribute`` option.
 
-    validates :zipcode, zipcode: { country_code: :es }
+```ruby
+validates :zipcode, zipcode: { country_code: :es }
 
-    validates :zipcode, zipcode: { country_code_attribute: :my_country_code_column }
+validates :zipcode, zipcode: { country_code_attribute: :my_country_code_column }
+```
 
 If you need to localize the error message, just add this to your I18n locale file:
 
-    errors:
-      messages:
-        invalid_zipcode: Your zipcode error message.
+```yaml
+errors:
+  messages:
+    invalid_zipcode: Your zipcode error message.
+```
+
+### Without ActiveModel::Validations
+
+```ruby
+ValidatesZipcode.valid?('93108', 'ES')
+# => true
+```
+
+### Formatting
+
+This gem can also be used for formatting zipcodes according to country specific rules.
```

Comment:
Thanks for adding all this to README. 

Please, add a note marking the countries supported, so we no mislead others.

Reply:
I've tried to point to the file with the transformations to make it clear.